staging_service/autodetect/GenerateMappings.py

```diff
@@ -0,0 +1,303 @@
+"""
+* This script generates various potential api responses and serves as a list of supported apps and extensions
+for the staging service endpoint .
+* Afterwards, we can pick a json or yaml file and edit those, or keep editing this file in the future to generate the mappings
+
+
+================================
+# In Scope Importer Apps
+* Note: FastQ Interleaved/Uninterleaved custom UIs not yet implemented
+* Note: MergeMetabolicAnnotations and PredictPhenotypes also require an object in addition to a file
+
+================================
+# Out of scope importer notes
+* Note: MultipleSequenceAlignmentApp Commented out because it has a weird interface
+
+================================
+# Batch App Importer notes
+* Commented out all batch apps, as those take a directory from the narrative itself
+
+Functionality: Running this script will
+* Save this indirectly as a json file
+* Save this indirectly as a yaml file
```

Comment:
it only saves as JSON at the moment - there's no yaml writer

Reply:
Thanks, I deleted that part but forgot to update the docs